__tests__/lib/github.js

```diff
@@ -6,6 +6,10 @@ const mockRepo = require('../utils').mockRepo;
 const parse = require('../../src/parsers/javascript');
 const github = require('../../src/github');
 
+// mock-fs is causing some unusual side effects with jest-resolve
```

Comment:
Would appreciate a close look at this. Is this okay temporarily?

Reply:
Sure, I can suss out the mock-fs details if the rest looks fine. It's tricky in a 'nesting git repositories' sense, but I can think of some workarounds, like instead of mocking these fs calls storing the files and writing them temporarily for the tests to access 'for real'.